content/terraform-plugin-framework/v1.15.x/docs/plugin/framework/resources/identity.mdx

```diff
@@ -0,0 +1,462 @@
+---
+page_title: Identity
+description: >-
+Learn how to implement resource identity in the Terraform plugin framework.
+---
+
+# Resource identity
+
+A resource identity is a data object that is stored alongside the resource instance state to refer to at most one remote object, across all instances of that provider.
+
+## Resource Identity Schema
+
+Practitioners can create an resource identity schema, like the existing managed resource schema, that consists of a map of attribute names and associated behaviours. With a few key differences:
+
+1. The identity schema will only support primitive types and list types such as:
+    - `BoolAttribute`
+    - `Float32Attribute`
+    - `Float64Attribute`
+    - `Int32Attribute`
+    - `Int64Attribute`
+    - `NumberAttribute`
+    - `StringAttribute`
+    - `ListAttribute`
+        - `ElementType: types.BoolType`
+        - `ElementType: types.Float32Type`
+        - `ElementType: types.Float64Type`
+        - `ElementType: types.Int32Type`
+        - `ElementType: types.Int64Type`
+        - `ElementType: types.NumberType`
+        - `ElementType: types.StringType`
+
+NOTE: The ordering in lists is expected to be stable and semantically relevant.
+
+2. Unlike the resource config schema, the resource identity schema will not support behaviours such as Required and Computed. Two new, but similar, behaviours will be allowed for each attribute: `RequiredForImport` and `OptionalForImport`.
+ - `RequiredForImport` is set to true if an import block would need this information as part of the identity attribute to import the resource. Some attributes might be specified through other means e.g. the provider configuration and therefore won't be needed for import.
+
+  - `OptionalForImport` is set to true if an import block can provide this information as part of the identity attribute to import the resource. This can be data that would normally be the default of the configured provider running the import. This does not mean the data is omitted from the resource identity, import-only data is not supported.
```

Comment:
Rewording, also the supporting of storing `null` for an identity attribute means that import only is technically possible 😆 

```suggestion
  - `OptionalForImport` is set to true if an import block does not require this information from the practitioner to import the resource. This can be data that would normally be defaulted from the configured provider running the import.
```